### PR TITLE
Address layout width issue when dashboard is on

### PR DESCRIPTION
**Pull request by sorta**
Intended to address https://github.com/travis-pro/team-teal/issues/2962

### CODE REVIEW

**Comment by sorta** [comment]
I was prepared for the line I removed to break the dashboard layout, but it doesn't seem to have done that. I've tested various parts of the app, and everything seems fine. I've also looked through the code to try to identify anything else that can break from this change, but haven't found anything. Seems like maybe adding the `.centered, .centered.wrapper ` selectors made the `.feature-wrapper.dashboard` selector superfluous.

**Comment by sorta** [comment]
Ahh there it is! of course I look at all sorts of stuff and miss one very obvious thing haha. I'll take a look at fixing the header

File: app/styles/app/layout.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-$max-width: 1024px;
+$max-width: $layout-max-width;
```
Comment (artursmirnov) [inline app/styles/app/layout.scss]: 👍

**Comment by artursmirnov** [comment]
@sorta It looks great to me, thank you. I spent some time testing it and didn't find any regressions, great job 👍

**Comment by sorta** [comment]
Excellent, thanks for the review and testing @artursmirnov :smile:

**Pull request merged**